lib/jobs/ScheduleManager.js

```diff
@@ -53,6 +56,22 @@ class ScheduleManager {
     return `${min} ${hr} * * *`;
   }
 
+  static getRandomHourlySchedule(interval) {
+    try {
+      const everyXhrs = parseInt(/^[0-9]+/.exec(interval)[0]);
+      logger.info(`schedule is to run every ${everyXhrs} hours`);
```

Comment:
it is better to put a not 'NaN' check before logging. Otherwise in case of input 'blah hours' log --> 'schedule is to run every blah hours' doesn't look good. if everyHhrs === NaN directly throw exception.

Reply:
@sauravmndl the logger will not be printed. It will throw exception at LN61 already as the regex wont be evaulated and it leads to accessing 0th index of an non-existing array. I had put an assert but that seemed unnecessary when testing.

Reply:
@sauravmndl  - one more thing, dont approve the PRs even if there are minor comments, let them be answered and then you can approve, else you would then leave it upto the committer :-)